src/index.ts

```diff
@@ -26,6 +26,13 @@ function createWindow() {
     return { action: 'deny' }
   })
 
+  win.webContents.on('will-navigate', (event, url) => {
+    if(!url.startsWith('https://app.graphite.dev')) {
```

Comment:
Why do we care whether this is a Graphite URL or a non-Graphite URL? I'd expect that we'd want all links to open in the same window?